packages/@react-types/shared/src/dom.d.ts

```diff
@@ -90,3 +90,8 @@ export interface TextInputDOMProps extends DOMProps {
   onBeforeInput?: FormEventHandler<HTMLInputElement>,
   onInput?: FormEventHandler<HTMLInputElement>
 }
+
+// A set of props allowed on SVG components
+export interface SVGProps {
```

Comment:
Is there a different way we should add the SVG props we want supported? SVGProps<T> conflicts with our DOMprops?